icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2658,6 +2682,13 @@ class U_I18N_API FormattedNumber : public UMemory, public FormattedValue {
      * @draft ICU 68
      */
     MeasureUnit getOutputUnit(UErrorCode& status) const;
+
+    /**
+     * The gender of the formatted output.
+     *
```

Comment:
start the comment with verb in form "starts, gets ... etc)

Reply:
Done.